src/main/kotlin/racingcar/InputValidator.kt

```diff
@@ -0,0 +1,10 @@
+package racingcar
+
+class InputValidator {
+    companion object {
+        fun checkNaturalNumber(input: String) {
+            val number = input.toIntOrNull()
+            require((number != null) && (number > 0))
```

Comment:
```suggestion
            require(input.toInt() > 0) { "1이상 정수로 입력해주세요" }
```

`require` 만으로도 충분할 것 같습니다. 단 `require`를 쓰면 에러 메시지 추가 부탁드립니다.